TRN-16/index.php

```diff
@@ -0,0 +1,33 @@
+
+<!DOCTYPE HTML>  
+<html>
+<head>
+<style>
+.error {color: #FF0000;}
+</style>
+</head>
+<body>  
+
+
+<p><span class="error">* required field</span></p>
+<form method="post" action="<?php echo htmlspecialchars($_SERVER['PHP_SELF']);?>">  
```

Comment:
what is this php self update

Reply:
This is used to get reload the page into same file. $SERVER['PHP_SELF']  returns the file name of current executing directory. This whole form tag will post action to the same file and the same file will be received again. And htmspecialchars will restrict the php from treating special characters as html syntax.